README.md

```diff
@@ -1,3 +1,6 @@
+## API documentation (Swagger)
+Open [http://localhost:3000/api/docs](http://localhost:3000/api/docs) to view it in the browser.
+
 ## Available Scripts
```

Comment:
How to run tests and setup test db?

Reply:
Run tests: https://github.com/HF5-PROJECT/api/blob/33feb9d5ffa3fd192173d50e9db1c44f5938e09d/README.md?plain=1#L17-L19

Setup DB:
https://github.com/HF5-PROJECT/api/blob/33feb9d5ffa3fd192173d50e9db1c44f5938e09d/README.md?plain=1#L25-L27